### ISSUE TITLE
trouble with config file homebrew; mac Sierra; previously mac El Capitan

### ISSUE THREAD

**Issue by MichaelDimmitt** [PromyLOPh/pianobar#609]
Having trouble on a mac with homebrew trying to make a config file. 
I can create the config file and make changes but am having problems following the manual. Because it is telling me those locations do not exist. And saving it in the spot where I find config example has no effect. 
Operating system: OSX sierra, previously was el - capitan

".config/pianobar: No such file or directory"
also,
"/.config/pianobar/config: No such file or directory"
I found the config example inside 

/usr/local/Cellar/pianobar/2015.11.22_2/contrib

similar to issue #101 

I even (cd .. ) until I am looking at all files on the computer.
and run the command 
'MacBook-Pro-3:/ compName$ find ./config/pianobar'

resultant output: 
`find: ./config/pianobar/: No such file or directory

Simple enough to change the config file so pianobar will run using my username and password. 
where do I put the config file?

<pre>/usr/local/Cellar/pianobar/2016.06.02/contrib</pre>
did not know what this was so did not put configuration file here. /usr/local/Cellar/libao/1.2.0/lib

basically I take the config-example file... cp it with same name "config" vi into it. Make changes putting in username and password. Save. file is now in location ... /usr/local/Cellar/pianobar/2016.06.02/contrib

I cannot move it to the location described in man. 
finally, I open pianobar in the terminal to open the application and ... it is prompting for the username and password. So config file I made is not having an effect.

Hopefully this is clear. After spending 4 hours trying to figure this out I thought I would post it as an issue. To see if other people are having the problem and know the fix. Because the man page did not seem to know the solution.

<b>version</b>
was at piano_bar 2015.11.22_2/
uninstalled and reinstalled now at: piano_bar 2016.06.02/

**Comment by PromyLOPh** [comment]
Just create the directory and move your config file there: `mkdir -p ~/.config/pianobar`

**Comment by MichaelDimmitt** [comment]
Worked Great!
